### ISSUE TITLE
swoole 4.2.1 open hook sleep  

### ISSUE THREAD

**Issue by mangowong** [swoole/swoole-src#1987]
```php
<?php

set_error_handler(function (int $errno, string $errstr, string $errfile, int $errline) {
    throw new Exception($errstr, $errno);
});
try {
    Swoole\Coroutine::create(function () {
        Swoole\Runtime::enableCoroutine(true);
        usleep(1); // less than 1000 will has problem
    });
    // usleep(1); //not in coroutine runtime will  coredump
} catch (\Throwable $throwable) {
    echo $throwable->getMessage().PHP_EOL;
}

```

**Comment by twose** [comment]
It's an experimental feature, we have some args to enable or disable the part of the hook
Thank you very much for your feedback, we will fix it soon.

**Comment by twose** [comment]
Fixed via 04eddd5
